lib/rets/metadata/rets_class.rb

```diff
@@ -28,11 +29,12 @@ def self.builds_tables(table_container, resource_id, lookup_types)
       def self.build(rets_class_fragment, resource_id, lookup_types, metadata)
         class_name = rets_class_fragment["ClassName"]
         visible_name = rets_class_fragment["VisibleName"]
+        standard_name = rets_class_fragment["StandardName"]
         description = rets_class_fragment["Description"]
 
         table_container = find_table_container(metadata, resource_id, class_name)
         tables = builds_tables(table_container, resource_id, lookup_types)
-        new(class_name, visible_name, description, tables)
+        new(class_name, visible_name, standard_name, description, tables)
```

Comment:
This part of the diff looks great, I'd be happy to merge as is if you wanted to separate it from the rescue block above.